.eslintignore

```diff
@@ -9,3 +9,4 @@ artifacts
 tmp
```

Comment:
Tried running `pnpm run check` ... unfortunately I got a JS OOM error.  Mostly-fresh everything, fresh `pnpm install` and ran both `clean:build` and `clean:cache`.
Then I did a "nuke"  (`git clean -xdf`) and now everything is passing.

So all this to say ... there's a hiccup somewhere, and we should keep our eyes out for it.

Reply:
OK! Good not know! The only OOM I've gotten so far was the storybook thing that you successfully diagnosed and fixed.